airbyte/mcp/_local_ops.py

```diff
@@ -63,7 +64,7 @@ def validate_connector_config(
     Returns a tuple of (is_valid: bool, message: str).
     """
     try:
-        source = get_source(connector_name)
+        source = get_source(connector_name, docker_image=is_docker_installed())
```

Comment:
Let's make slightly more explicit. When docker installed, `True`, otherwise, explicit `False`.

```suggestion
        source = get_source(
            connector_name,
            docker_image=is_docker_installed() or False,
        )
```

I'll apply. Pull in a sec.